armadillo1/README.md

```diff
@@ -38,128 +56,53 @@ Your requirements are as follows
-              environment: "{{ rserve.environments[0].name }}"
-              whitelist:
-                - dsBase
-              # options:
-                # datashield:
-                # override default DataSHIELD options
-    - role: molgenis.armadillo.service_minio
-      vars:
-        version: 2022-01-25T19-56-04Z
-        data: /var/lib/minio/data
-        protocol: http
-        domain: "{{ minio.domain }}"
-        root_user: "{{ minio.root_user }}"
-        root_password: "{{ minio.root_password }}"
-        console:
-          version: 0.14.8
-    - role: molgenis.armadillo.service_auth
-      vars:
-        image: 
-          version: 0.3.2
-          repo: molgenis
-          name: molgenis-auth
-        api_token: xxxxxxxxxxxxxxxxx
-        base_url: http://armadillo-auth.local
-    - role: molgenis.armadillo.service_rserve
-    - role: molgenis.armadillo.post_assemble
-    - role: molgenis.armadillo.post_nginx
+        enabled: true
+...
+        domain: cohort-storage-console.armadillo.domain.org
+...
+    - role: post_nginx
       vars:
         enabled: true
         domains: 
-          armadillo: armadillo.local
+          armadillo: cohort.armadillo.domain.org
           storage: "{{ minio.domain }}"
           console: "{{ minio.console.domain }}"
-          auth: armadillo-auth.local
-        letsencrypt:
-          enabled: false
-          acme:
-            email: user@example.org
-    - role: molgenis.armadillo.tools_upgrade
+          auth: cohort-auth.armadillo.domain.org
 ```
 
-There are a few prerequisites that we need. 
-
-### "become" needs to work
-When you login to a VM you are hopefully yourself as in a useraccount that is recognisable as your account. After you logged in you need to be able to perform `sudo su` without entering a password. Get that in place and you will be able to run the playbook.
+> Keep in mind that when you see something like this: `storage: "{{ minio.domain }}"` it will be filled in automatically. You don't have to change it.
 
 ### Authentication and authorisation
-Before you deploy you need to register your application on the DataSHIELD authentication server. This allows you to delegate the authentication and usermanagement. The authorisation will still be under your control.
+Before you deploy the Armadillo stack you need to register your stack on the DataSHIELD authentication server. This allows you to delegate the authentication and user management. The authorisation will still be under your control(who gets access and who don't have access). To registrate you will need to send a mail to `molgenis-support@umcg.nl` with the chosen domains for the authentication, Minio and Minio console applications. Also add to the mail that you want to register for the the DataSHIELD authentication server and if you belong to a project like Lifecycle, Athlete or Longitools. When the Armadillo stack is registrerd you will get an mail back with data that need to be inserted below. 
```

Comment:
Het lijkt mij goed dat dit stuk test bovenin word gezet. Zodat het duidelijk is dat dit een prerequisite is voor het project.